packages/lesswrong/components/users/SocialMediaLink.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { combineUrls, registerComponent } from "../../lib/vulcan-lib";
+import { socialMediaIconPaths } from "../form-components/PrefixedInput";
```

Comment:
Yeah - it didn't seem worth the effort since this is the only place they're used. There's also some nice typing around only allowing existing social media platforms that would probably be lost (or trickier) if we used ForumIcon.